### DISCUSSION TITLE
Generate from TAGS

### DISCUSSION THREAD

**Discussion by mateusfg7** [BobAnkh/auto-generate-changelog#106] [Q&A] [answered]
There is a way to generate changeling based on the last repository tag, instead of last release?

**Comment by BobAnkh** [comment, marked as answer]
Sorry, doesn't support this feature at this time. This is because the action can only choose tags or releases as the reference. In this action, we collect all the releases from GitHub API and analyze the commits. You can use other tools for this requirement.
